packages/Webkul/Shop/src/Http/Controllers/CartController.php

```diff
@@ -24,24 +25,35 @@ class CartController extends Controller
      */
     protected $productRepository;
 
+    /**
+     * OrderrRepository object
+     *
+     * @var \Webkul\Sales\Repositories\OrderRepository
+     */
+    protected $orderRepository;
+
     /**
      * Create a new controller instance.
      *
      * @param  \Webkul\Customer\Repositories\CartItemRepository  $wishlistRepository
      * @param  \Webkul\Product\Repositories\ProductRepository  $productRepository
+     * @param  \Webkul\Order\Repositories\OrderRepository  $orderRepository
      * @return void
      */
     public function __construct(
         WishlistRepository $wishlistRepository,
-        ProductRepository $productRepository
+        ProductRepository $productRepository,
+        OrderRepository $orderRepository
     )
     {
-        $this->middleware('customer')->only(['moveToWishlist']);
+        $this->middleware('customer')->only(['moveToWishlist','reorder']);
```

Comment:
Use space after and before with binary operator

Reply:
done